### PR TITLE
[web][api][SIMS #1300]PIR moved to institutions API client

### PR DESCRIPTION
**Pull request by andrewsignori-aot**
- Program information request related methods moved to a specific institution client controller.
- Changed PIR get data to rely always on the original assessment instead of the current assessment to ensure that every time that the PIR UI is shown the data will be coming from the original assessment record.

### CODE REVIEW

File: sources/packages/api/src/services/application/application.service.ts [hunk 1]
```diff
@@ -428,20 +430,31 @@ export class ApplicationService extends RecordDataModelService<Application> {
...
+        "studentAssessments.triggerType",
       ])
       .innerJoin("application.programYear", "programYear")
       .leftJoin("application.pirProgram", "pirProgram")
       .innerJoin("application.student", "student")
+      .innerJoin("student.sinValidation", "sinValidation")
       .innerJoin("application.location", "location")
-      .innerJoin("application.currentAssessment", "currentAssessment")
-      .leftJoin("currentAssessment.offering", "offering")
+      .innerJoin("application.studentAssessments", "studentAssessments")
```
Comment (ann-aot) [inline sources/packages/api/src/services/application/application.service.ts:443]: 👍

**Review by henryoforeh-aot — APPROVED** [review]

File: sources/packages/api/src/route-controllers/program-info-request/program-info-request.institutions.controller.ts [hunk 2]
```diff
@@ -284,16 +275,16 @@ export class ProgramInfoRequestController extends BaseController {
...
-   * @returns student application list of an institution location
+   * @returns student application list of an institution location.
    */
   @HasLocationAccess("locationId")
   @Get(":locationId/program-info-request")
   async getPIRSummary(
-    @Param("locationId") locationId: number,
-  ): Promise<PIRSummaryDTO[]> {
+    @Param("locationId", ParseIntPipe) locationId: number,
+  ): Promise<PIRSummaryAPIOutDTO[]> {
```
Comment (ann-aot) [inline sources/packages/api/src/route-controllers/program-info-request/program-info-request.institutions.controller.ts:287]: a qn, are we doing date refactoring as part of the story, `studyStartPeriod: getISODateOnlyString(offering?.studyStartDate),
        studyEndPeriod: getISODateOnlyString(offering?.studyEndDate),`

**Review by ann-aot — APPROVED** [review]
Good  Work @andrewsignori-aot. 👍 just a small qn

**Review by dheepak-aot — APPROVED** [review]
LGTM @andrewsignori-aot

**Review by guru-aot — APPROVED** [review]
LGTM

File: sources/packages/api/src/route-controllers/program-info-request/program-info-request.institutions.controller.ts (hunk 2 above)
Comment (andrewsignori-aot) [reply sources/packages/api/src/route-controllers/program-info-request/program-info-request.institutions.controller.ts:287]: Date-only values should be sent from the API in ISO format.

**Pull request merged**